lib/ProMotion/table/extensions/searchable.rb

```diff
@@ -11,6 +11,8 @@ def make_searchable(params={})
           search_bar.placeholder = params[:search_bar][:placeholder]
         end
 
+        @no_results_text = params[:search_bar][:no_results] if params[:search_bar][:no_results]
```

Comment:
we probably dont need to guard this, right?  If its nil, then it will just be set to nil, which is the same as not setting it